docs/debugging/checkify_guide.md

```diff
@@ -251,7 +251,42 @@ jax.grad(assert_gradient_negative)(-1.)
 # ValueError: gradient needs to be negative!
 ```
 
-<!-- TODO: scan? -->
+### `scan`/`while_loop`/`cond`
+
+Checkify works with all higher-order control flow operators, like `scan`,
```

Comment:
Can you make these into cross-references? I forget the markdown syntax for it; in restructured text it is :func:`~jax.lax.scan`
Similarly for while_loop and cond.

Reply:
I think it's
```
{func}`jax.lax.scan`
```
in markdown.